### PR TITLE
Git&Build Tools theory

### PR DESCRIPTION
**Pull request by Lizaveta-CR**

### CODE REVIEW

**Review by Toxa1987 — APPROVED** [review]
Looks good for me.

File: stage #2/module #1. GIT & Build Tools/build tools/build_tools.md [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+
+## Materials
+
++ [Gradle Getting Started](https://docs.gradle.org/current/userguide/getting_started.html)
++ [Maven documentation](https://maven.apache.org/guides/)
+
+### Intro
+
+Software development is a complex process. A lot of people can work on it.
+Before we can see the application it needs to be packaged into a binary executable.The application itself may (and always will) have dependencies (e.g. external JAR files from
```
Comment (AliakseiYermachonak) [inline stage #2/module #1. GIT & Build Tools/build tools/build_tools.md:17]: executable.The - missing space

File: stage #2/module #1. GIT & Build Tools/build tools/build_tools.md [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+             ...
+           </project>
+        ```
+
+      A Maven plugin is a group of goals; however, these goals aren't necessarily all bound to the same phase.
+
+### Gradle Basics
+
+Gradle is a build automation system that is fully open source and uses the concepts you see on Apache Maven and Apache Ant. It uses domain-specific language based on the
+programming language Groovy,
```
Comment (AliakseiYermachonak) [inline stage #2/module #1. GIT & Build Tools/build tools/build_tools.md:251]: or Koltin

**Review by AliakseiYermachonak — APPROVED** [review]

File: stage #2/module #1. GIT & Build Tools/build tools/build_tools.md (hunk 1 above)
Comment (BogdanovaDiana) [reply stage #2/module #1. GIT & Build Tools/build tools/build_tools.md]: Thank you, fixed

File: stage #2/module #1. GIT & Build Tools/build tools/build_tools.md (hunk 2 above)
Comment (BogdanovaDiana) [reply stage #2/module #1. GIT & Build Tools/build tools/build_tools.md]: Thank you, fixed

**Pull request merged**